conf/app.routes

```diff
@@ -247,3 +247,10 @@ GET         /:taxYear/:businessId/expenses/change-repairs-maintenance/disallowab
 POST        /:taxYear/:businessId/expenses/change-repairs-maintenance/disallowable-amount        controllers.journeys.expenses.repairsandmaintenance.RepairsAndMaintenanceDisallowableAmountController.onSubmit(taxYear: Int, businessId: String, mode: Mode = CheckMode)
 
 GET         /:taxYear/:businessId/expenses/repairs-maintenance/check                             controllers.journeys.expenses.repairsandmaintenance.RepairsAndMaintenanceCostsCYAController.onPageLoad(taxYear: Int, businessId: String)
+
+########## Expenses - Entertainment ##########
```

Comment:
Tax year and business id can be parsed to their richer type from the routes. See routes on line 239.